test/plugin/test_in_snmptrapalert.rb

```diff
@@ -2,77 +2,107 @@
 
 class SnmptrapalertInputTest < Test::Unit::TestCase
 
-    setup do
-        Fluent::Test.setup
-    end
+  setup do
+    Fluent::Test.setup
+  end
+
+  SNMP_CONFIG = %[
+    host 0.0.0.0
+    port 162
```

Comment:
Why are you reverting this to 162? I thought it had to be changed to 1620 to be able to run the tests.